source/_integrations/unifiprotect.markdown

```diff
@@ -0,0 +1,88 @@
+---
+title: UniFi Protect
+description: Instructions on how to configure UniFi Protect integration by Ubiquiti.
+ha_category:
+  - Hub
+  - Camera
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_quality_scale: platinum
+ha_codeowners:
+  - '@briis'
+  - '@AngellusMortis'
+  - '@bdraco'
+ha_domain: unifiprotect
+ha_platforms:
+  - camera
+---
+
+The [UniFi Protect Integration](https://ui.com/camera-security) by [Ubiquiti Networks, inc.](https://www.ui.com/), adds support for retrieving Camera feeds and Sensor data from a UniFi Protect installation on either a UniFi OS Console running UniFi Protect.
+
+{% include integrations/config_flow.md %}
+
+## Device Support
+
+All known UniFi Protect devices are should be supported. Each device will get a variety of entities added for each of the different entity platforms.
+
+### Camera
+
+A camera entity for each camera channel and RTSP(S) combination found on the NVR device will be created. Only the highest resolution RTSPS camera entity will be enabled by default. Changing your RTSP settings in UniFi Protect will require you to reload the `unifiprotect` integration or restart Home Assistant.
```

Comment:
Single line chapter? With mixed content. It contains something about the camera's RTSP (which isn't a thing an end-user has to deal with?), and contains instructions on restarting. `uinifiprotect` isn't code, so shouldn't be code fenced, and restarting Home Assistant is not needed at all.

Reply:
This was originally a bullet list on the HACS README. One per entity platform. I was thinking a header made it look a bit better (reference for how the original looked: https://github.com/briis/unifiprotect/)

Reply:
It doesn't make sense on this page IMHO. There is trouble shooting, something out a protocol they shouldn't care about, entities, channels, restarting integrations and Home assistant... all in that single line. That is out of focus and out of place IMHO.

Reply:
The bit about reloading/restarting was new. I can remove that though. But this section is designed to just list all of the things the integration adds. So right now, it just had the single header for the camera platform as that is all that is in the initial PR, but it will be longer. 

If you do not think this section is useful, I can just scrap it all together.

Reply:
Reworked this so it is a bullet list of added entities per device type (camera, light, sensor, viewport, NVR).